### DISCUSSION TITLE
Is noDRM able to de-DRM Kindle books that have VOUCHER file in it?

### DISCUSSION THREAD

**Discussion by p1nkyy** [noDRM/DeDRM_tools#562] [Q&A] [answered]
I get most of my Kindle files from libraries and 90% of the time they always get de-DRM-ed perfectly well. But after January 2023, some books I noticed has VOUCHER file in it and it can't be de-DRM-ed. Is it possible for them to be de-DRM-ed at all?

https://puu.sh/K9hz9/e4d22d84c3.png

FWIW, I'm using:
Win 10
Calibre 7.12
NoDRM 10.0.3

**Comment by ElleKayEm** [comment, marked as answer]
Possibly with DeDRM 10.0.9 and the KFX Input plugin.

**Reply by p1nkyy** [reply]
Thanks, it indeed works!
